### PR TITLE
ocamlPackages.parmap: init at 1.1

### PR DESCRIPTION
**Pull request by bcdarwin**
###### Motivation for this change

Add a package.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [NA] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [NA] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/ocaml-modules/parmap/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    sha256 = "0xpy2012i8ckpii4cqxwcr5v7d4719md7y6rfdqyv2xy8nq0qhrf";
+  };
+
+  doCheck = true;
+
+  meta = with lib; {
+    description = "Library for multicore parallel programming";
+    homepage = "https://rdicosmo.github.io/parmap";
+    maintainers = [ maintainers.bcdarwin ];
+    license = licenses.gpl2;
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/parmap/default.nix]: Isn’t it `lgpl2`?

**Review by vbgl — CHANGES_REQUESTED** [review]

File: pkgs/development/ocaml-modules/parmap/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+buildDunePackage rec {
+  pname = "parmap";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "rdicosmo";
+    repo   = pname;
+    rev    = version;
+    sha256 = "0xpy2012i8ckpii4cqxwcr5v7d4719md7y6rfdqyv2xy8nq0qhrf";
+  };
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/parmap/default.nix]: Please use the release archive from <https://github.com/rdicosmo/parmap/releases/download/1.1/parmap-1.1.tbz>. It has correct version information.

File: pkgs/development/ocaml-modules/parmap/default.nix (hunk 1 above)
Comment (bcdarwin) [reply pkgs/development/ocaml-modules/parmap/default.nix]: Indeed, fixed.

File: pkgs/development/ocaml-modules/parmap/default.nix (hunk 2 above)
Comment (bcdarwin) [reply pkgs/development/ocaml-modules/parmap/default.nix]: OK, done.

**Comment by vbgl** [comment]
@GrahamcOfBorg build ocamlPackages.parmap

**Review by vbgl — APPROVED** [review]

**Pull request merged**